### ISSUE TITLE
Exception while using a _Visitor to visit a FieldDeclarationImplin ClassDeclarationImplin CompilationUnitImpl

### ISSUE THREAD

**Issue by devoncarew** [dart-lang/linter#646]
Looks like this is in the `overridden_fields` lint. Here's a partial stack trace:

```
RangeError (index): Invalid value: Valid value range is empty: 0

#0 _StringBase.[] (dart:core-patch/string_patch.dart:229)
#1 _Visitor._getOverriddenMember.isOverriddenMember (package:linter/src/rules/overridden_fields.dart:123)
#2 Object&ListMixin.any (dart:collection/list.dart:115)
#3 _Visitor._getOverriddenMember.containsOverriddenMember (package:linter/src/rules/overridden_fields.dart:131)
#4 Iterable.firstWhere (dart:core/iterable.dart:520)
#5 _Visitor._getOver...
```

**Comment by scheglov** [comment]
I think this is already fixed.
https://github.com/dart-lang/linter/commit/22c999e62f4894fabd09289d0a5b91830a283999

**Comment by devoncarew** [comment]
Awesome! @pq, do we need to do anything to get this fix rolled into the sdk, or has that already happened? (it would just be a DEP update, correct?)

**Comment by pq** [comment]
@devoncarew : this will require a `DEPS` bump.  Ideally I'd like to roll `0.1.31` (https://github.com/dart-lang/linter/issues/494) and update the SDK to pull it in.  In a pinch we can cherry-pick 22c999e into a patch release.

@whesse : are we on track for a dev roll in the Wednesday time-frame?

**Comment by whesse** [comment]
Yes, we will do a dev roll then.  I'm not sure if it is needed before then.  I'm not certain if that will be the last full push to dev.

**Comment by pq** [comment]
Thanks @whesse !

@devoncarew : shooting to push a release today or tomorrow (https://github.com/dart-lang/linter/issues/494).

**Comment by pq** [comment]
Closing; fixed w/ 22c999e.
